src/Faker/Provider/pl_PL/LicensePlate.php

```diff
@@ -0,0 +1,542 @@
+            'Olsztyn' => ['O'],
+            'bartoszycki' => ['BA'],
+            'braniewski' => ['BR'],
+            'działdowski' => ['DZ'],
+            'elbląski' => ['EB'],
+            'ełcki' => ['EL'],
+            'giżycki' => ['GI'],
+            'iławski' => ['IL'],
+            'kętrzyński' => ['KE'],
+            'lidzbarski' => ['LI'],
+            'mrągowski' => ['MR'],
+            'nidzicki' => ['NI'],
+            'nowomiejski' => ['NM'],
+            'olecki' => ['OE'],
+            'gołdapski' => ['GO'],
+            'olsztyński' => ['OL'],
+            'ostródzki' => ['OS'],
+            'piski' => ['PI'],
+            'szczycieński' => ['SZ'],
+            'węgorzewski' => ['WE'],
+        ],
+        'P' => [
+            'Kalisz' => ['A', 'K'],
+            'Konin' => ['KO', 'N'],
+            'Leszno' => ['L'],
+            'Poznań' => ['O', 'Y'],
+            'chodzieski' => ['CH'],
+            'czarnkowsko-trzcianecki' => ['CT'],
+            'gnieźnieński' => ['GN'],
+            'gostyński' => ['GS'],
+            'grodziski' => ['GO'],
+            'jarociński' => ['JA'],
+            'kaliski' => ['KA'],
+            'kępiński' => ['KE'],
+            'kolski' => ['KL'],
+            'koniński' => ['KN'],
+            'kościański' => ['KS'],
+            'krotoszyński' => ['KR'],
+            'leszczyński' => ['LE'],
+            'międzychodzki' => ['MI'],
+            'nowotomyski' => ['NT'],
+            'obornicki' => ['OB'],
+            'ostrowski' => ['OS'],
+            'ostrzeszowski' => ['OT'],
+            'pilski' => ['P'],
+            'pleszewski' => ['PL'],
+            'poznański' => ['OZ', 'Z'],
+            'rawicki' => ['RA'],
+            'słupecki' => ['SL'],
+            'szamotulski' => ['SZ'],
+            'średzki' => ['SR'],
+            'śremski' => ['SE'],
+            'turecki' => ['TU'],
+            'wągrowiecki' => ['WA'],
+            'wolsztyński' => ['WL'],
+            'wrzesiński' => ['WR'],
+            'złotowski' => ['ZL'],
+        ],
+        'Z' => [
+            'Koszalin' => ['K'],
+            'Szczecin' => ['S', 'Z'],
+            'Świnoujście' => ['SW'],
+            'białogardzki' => ['BI'],
+            'choszczeński' => ['CH'],
+            'drawski' => ['DR'],
+            'goleniowski' => ['GL'],
+            'gryficki' => ['GY'],
+            'gryfiński' => ['GR'],
+            'kamieński' => ['KA'],
+            'kołobrzeski' => ['KL'],
+            'koszaliński' => ['KO'],
+            'łobeski' => ['LO'],
+            'myśliborski' => ['MY'],
+            'policki' => ['PL'],
+            'pyrzycki' => ['PY'],
+            'sławieński' => ['SL'],
+            'stargardzki' => ['ST'],
+            'szczecinecki' => ['SZ'],
+            'świdwiński' => ['SD'],
+            'wałecki' => ['WA'],
+        ],
+        'U' => [
+            'Siły Zbrojne Rzeczypospolitej Polskiej' => ['A', 'B', 'C', 'D', 'E', 'G', 'I', 'J', 'K', 'L'],
+        ],
+        'H' => [
```

Comment:
Since these are special government-reserved licence plates, maybe we should add a flag to allow exclude these from generation (default should be excluded).

Reply:
What would you suggest?

Maybe just default first argument of `licensePlate` to the list of voivodeships only without `army` and `services` and create a separate method like `licensePlateWithSpecials` which would generate license plate with army and service plates?

Or I can add a 3rd argument with default false and if it's false then I can filter the voivodeships list to exclude `army` and `services`.

Reply:
3rd argument idea seems cleaner to me. @bramceulemans what do you think?

Reply:
I think a third argument would be the most clean option.

Reply:
So I've added it as the 1st option instead of 3rd. The reason is I think it would be more confusing to call `$faker->licensePlate(['army', 'services'], [])` and get standard plates assigned to voivodeships instead because the third argument defaults to null.

Also if I wanted to get all the license plates I would have to call `$faker->licensePlate([], [], true)` or `$faker->licensePlate(null, null, true)`, now you can simply call `$faker->licensePlate(true)`.

It's easy to move it to the 3rd place if needed.

Reply:
Sounds good. Personally I think `$voivodeships` and `$counties` would suffice as parameter names, btw.

Reply:
Changed and also had to update a typo in one voivodeship name.